### ISSUE TITLE
Add helm config variables for image versions

### ISSUE THREAD

**Issue by sivamu** [retaildevcrews/ngsa#112]
## Description

What: 
- Add helm variables for the NGSA, webv and logging side car images
 
Why:
- Improve developer experience to make it easy deploying applications with a specific version

When:
- Whenever a deploy is taking place

Where:
 - Any AKS cluster needing helm deployments

## Tasks
- [ ] Add helm variables for the NGSA and Log Sidecar image repo versions. https://github.com/retaildevcrews/ngsa/issues/117

## Acceptance Criteria

- Newly added helm variables should be in `helm-config.yaml` override files
- Allow dev to pass in custom image parameters in helm install
- Documentation should be complete

## Constraints
- N/A 

## References:
- References #43

**Comment by jofultz** [comment]
Will be replacing with Fluentd implementation #123.  May need to close this as no longer needed once #123 is complete.
